documentation/APIbacklog.md

```diff
@@ -788,27 +789,33 @@ This is a live doc that captures the status of all the APIs which have been form
       <th>Dynamic Predictive Connectivity Data</th>
       <td>Telefonica*<div></div></td>
       <td>
-        <a href="https://github.com/camaraproject/APIBacklog/pull/164">Template</a><div>2025/01/03</div>
+        <a href="https://github.com/camaraproject/APIBacklog/blob/main/documentation/API%20proposals/APIProposal_Dynamic%20Dynamic%20Predictive%20Connectivity%20Data_Telefonica.md">Template</a><div>2025/01/03</div>
       </td>
       <td><ul><li>TSC Approved (2025/03/06)</li></ul></td>
-      <td>Pending</td>
+      <td>
+        <a href="https://github.com/camaraproject/PredictiveConnectivityData">PredictiveConnectivityData</a>
+        <a href="https://github.com/camaraproject/PredictiveConnectivityData/blob/main/MAINTAINERS.MD">Maintainers</a>
+      </td>
       <td>N/A</td>
     </tr>
     <tr>
       <th>IoT Network Optimization</th>
       <td>TIM*<div></div></td>
       <td>
-        <a href="https://github.com/camaraproject/APIBacklog/pull/144">Template</a><div>2024/11/27</div>
+        <a href="https://github.com/camaraproject/APIBacklog/blob/main/documentation/API%20proposals/APIproposal_IoT_Data_Transfer_Activation_TIM.md">Template</a><div>2024/11/27</div>
       </td>
       <td><ul><li>TSC Approved (2025/03/06)</li></ul></td>
-      <td>Pending</td>
+      <td>
+        <a href="https://github.com/camaraproject/IoTNetworkOptimization">IoTNetworkOptimization</a>
+        <a href="https://github.com/camaraproject/IoTNetworkOptimization/blob/main/MAINTAINERS.MD">Maintainers</a>
+      </td>
       <td>N/A</td>
     </tr>
     <tr>
       <th>Network Traffic Analysis</th>
       <td>China Unicom*<div></div></td>
       <td>
-        <a href="https://github.com/camaraproject/APIBacklog/pull/129">Template</a><div>2024/11/19</div>
+        <a href="https://github.com/camaraproject/APIBacklog/blob/main/documentation/API%20proposals/APIproposal_NetworkTrafficAnalysis_chinaunicom.md">Template</a><div>2024/11/19</div>
       </td>
       <td><ul><li>Updates requested by TSC (2025/02/27)</li></ul></td>

```

Comment:
This proposal was approved by TSC on April 17th, there is already the Onboarding Tracker: https://github.com/camaraproject/APIBacklog/issues/202

Reply:
Thanks for taking my suggestion. And no, the TSC on April 3rd hasn't approved any new APIs, April 17th is correct, see https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/106528769/2025-04-17+TSC+Minutes